### ISSUE TITLE
encoding

### ISSUE THREAD

**Issue by pcubillos** [pcubillos/bibmanager#91]
As pointed out by @Serenade600  if you could put in your coding source some
```
encoding="utf-8"
```
right ahead the right brackets in
```
open(bibfile,****)
```
to encode with UTF-8, which finally become
```
open(bibfile,****,encoding="utf-8")
```
, I would much more appreciate it. In last two versions I had to add them myself every time after updating. I am making this suggestion as it does no harm to other functions but only add popularity with little work.

**Comment by pcubillos** [comment]
Hi @Serenade600
OK, I set the utf-8 encoding when opening text files. I think that I got all of them, but could you also check that these are the ones you referred to please?
https://github.com/pcubillos/bibmanager/pull/94/files

thanks

**Comment by Serenade600** [comment]
Hi @pcubillos , thanks for your effort, an amazing tool. The changes on _bib_manager.py_ and _config_manager.py_ is exactly what I referred to. I have not used the Latex Management function so far, but a high possibility that the modification also works well.

**Comment by pcubillos** [comment]
Great, thanks! merging in a minute!

**Comment by pcubillos** [comment]
Resolved in #94
